README.md

```diff
@@ -42,9 +42,13 @@ If you encounter a bug or need support   please see the official [README](https:
 
 This is expected behavior.  *Load Balanced Scheduler* balances cards that stock Anki would "fuzz".  Anki doesn't "fuzz" failed cards when they move back into the review queue so they are not load balanced. Additionally, while this would be easy to add, it would patch core parts of the v2 scheduler which is still in development.  It's not worth putting scheduling at risk if there is a silent failure.  The "fuzz" function itself has been very stable over time.
 
-### Under the experimental v2 scheduler logs show multiple intervals being calculated for the same card.
+### Under the v2 scheduler, logs show multiple intervals being calculated for the same card.
 
-This is expected behavior. Anki calculates intervals before checking which ease button the user actually pressed.  See [this function](https://github.com/dae/anki/blob/b5785f7ec8b3f95f88ba63cc43f9ee7ce829241a/anki/schedv2.py#L895-L915) in `schedv2.py` if you want to explore more.  The end result is that for a NORMAL card the log will show calculated intervals for HARD and NORMAL.  For an EASY cards the log will show calculated intervals for HARD, NORMAL and EASY. In the end, the last interval shown in the log is the one Anki assigns to the card.   
+This is expected behavior. Anki calculates intervals before checking which ease button the user actually pressed.  See [this function](https://github.com/ankitects/anki/blob/9f0929db32aaa9b589af20d7c8abb1a433e76cff/pylib/anki/scheduler/v2.py#L904-L925) in `scheduler/v2.py` if you want to explore more.  The end result is that for a NORMAL card the log will show calculated intervals for HARD and NORMAL.  For an EASY cards the log will show calculated intervals for HARD, NORMAL and EASY. In the end, the last interval shown in the log is the one Anki assigns to the card.
+
+### Filtered decks don't get load balanced.
+
+This is due to Anki not calling the "fuzz" function when using filtered decks. More info in [issue #5](https://github.com/xquercus/load-balanced-scheduler/issues/5#issuecomment-438806514).
```

Comment:
Perhaps this is outdated information, considering this comment is from 2018, but I've tested filtered decks on Anki v2.1.48 with this addon and it works fine.

I think we can safely remove this section